lighthouse-core/audits/byte-efficiency/uses-request-compression.js

```diff
@@ -40,8 +41,12 @@ class ResponsesAreCompressed extends ByteEfficiencyAudit {
   static audit_(artifacts) {
     const uncompressedResponses = artifacts.ResponseCompression;
 
+    function skipUnrelatedResponses(record) {
+      return URL.getProtocol(record.url) !== CHROME_EXTENSION_PROTOCOL;
```

Comment:
we recently discovered that calling `new URL` in a loop on the network records can be crazy slow due to data URIs (see #2701 for discussion), how about checking `record.protocol`/`record.scheme` or doing a simple `url.startsWith` if there's still some strong opinions about avoiding the network records 

cc: @brendankenny